src/context/VerifyContext.js

```diff
@@ -138,6 +144,21 @@ export function VerifyProvider(props) {
     //set loading status
     setIsLoading(true);
 
+    if (!filter.organizationId && orgId === null) {
```

Comment:
This looks like duplicate code (from `GrowerContext`). Given the complexity of the logic, it would definitely be worth making a common function that does this.